### DISCUSSION TITLE
S pen button

### DISCUSSION THREAD

**Discussion by ArcticJellyfish23** [biigle/core#814] [Ideas]
I am using a Samsung laptop with touch - which is working great with BIIGLE - and my S pen. Would it be possible to move through my favourite annotations by clicking the button on the pen? It would speed up the process a lot!

**Comment by dlangenk** [comment]
In case you didn't know, you can use the numbers 1,2,3,..,0 to switch between your favorites.

Since I think none of us has an s pen, it would be very hard to implement something like that. Probably having a hotkey on the keyboard to go from one favorite to the next could be implemented and you can likely have the s pen button send a keyboard press event.

**Reply by ArcticJellyfish23** [reply]
I see, thanks!
